trace/noop.go

```diff
@@ -7,16 +7,18 @@ import (
 	"context"
 
 	"go.opentelemetry.io/otel/trace"
+	"go.opentelemetry.io/otel/trace/noop"
 
 	"github.com/ava-labs/avalanchego/utils/constants"
 )
 
 var Noop Tracer = noOpTracer{
-	t: trace.NewNoopTracerProvider().Tracer(constants.AppName),
+	t: noop.NewTracerProvider().Tracer(constants.AppName),
```

Comment:
It seems like `noop.Tracer` already implements `Start` - so I think we can probably just get away with:

```golang
// noOpTracer is an implementation of Tracer that does nothing.
type noOpTracer struct {
	noop.Tracer
}

func (noOpTracer) Close() error {
	return nil
}
```

Thoughts?

Reply:
That's better